### PR TITLE
switch to webpack+fable-loader & fake5

### PR DESCRIPTION
**Pull request by humhei**
PR for #162

### CODE REVIEW

**Comment by humhei** [comment]
Appveyor CI interroped with 
`npm "whoami"`

**Comment by alfonsogarciacaro** [comment]
Any idea why CI is failing @ctaggart?

**Comment by ctaggart** [comment]
@alfonsogarciacaro No, not with just looking at the error.

**Comment by humhei** [comment]
@ctaggart  Now over to you,i don't known how to fix it 😕

File: tools/deploy.fsx [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+        node (toolDir</>"build-update.package.js")
+        let version = sprintf "0.6.0-build.%s" buildVersion
+        yarn <| sprintf "version --new-version %s --no-git-tag-version" version
+        npm "pack"
+        
+        let repoName = environVar "appveyor_repo_name"
+        let repoBranch = environVar "appveyor_repo_branch"
+        
+        if repoName = "fable-compiler/ts2fable" && repoBranch = "master" then
+            let line = sprintf "//registry.npmjs.org/:_authToken=%s\n" <| environVar "npmauthtoken`n"
```
Comment (alfonsogarciacaro) [inline tools/deploy.fsx]: Can you try removing \`n from `environVar "npmauthtoken"`?

File: tools/deploy.fsx [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+          "YarnInstall"
+          "Restore"
+          "BuildTest"
+          "RunTest" 
+          "BuildCli"
+          "RunCli"
+          "RestoreTestCompile"
+          "Publish" ]
+
+Target.RunOrDefault "Deploy"
```
Comment (alfonsogarciacaro) [inline tools/deploy.fsx]: With this the CI server will try to publish a new version to npm for every push (including PRs as it's happening now). @ctaggart how are you triggering deployments in Appveyor now?

Comment (humhei) [reply tools/deploy.fsx]: I collected all of appveyor server [Environments](https://ci.appveyor.com/project/fable-compiler/ts2fable/build/0.6.0-build.200)
And find the `APPVEYOR_PULL_REQUEST_HEAD_REPO_NAME` should be 
`fable-compiler/ts2fable`(current is `humhei/ts2fable`)

Comment (ctaggart) [reply tools/deploy.fsx]: Currently a successful build on master will use the `npmauthtoken` environment variable to publish. 
image

Comment (ctaggart) [reply tools/deploy.fsx]: Nevermind, looks like @humhei figured it out.

**Comment by alfonsogarciacaro** [comment]
Great, let's merge! Thanks a lot for your work! ❤️

**Pull request merged**